lib/blocks.js

```diff
@@ -29,20 +35,61 @@ var BLOCK_LIMIT = 200;
 BlockController.prototype.block = function(req, res, next, hash) {
```

Comment:
Do req, res, and next have to be the first 3 args? It seems like it would be kinda awkward to have the callback not be the last argument.